lucien-ios/Camera/CameraViewController.swift

```diff
@@ -9,14 +9,22 @@
 import UIKit
 import AVFoundation
 
-class CameraViewController: UIViewController, AVCaptureVideoDataOutputSampleBufferDelegate {
+class CameraViewController: UIViewController, AVCapturePhotoCaptureDelegate, Alertable {
+
+    // MARK: - Private IBOutlets
 
     @IBOutlet private weak var cameraView: UIView!
 
+    // MARK: - Variables
+
+    weak var cameraViewDelegate: CameraViewDelegate?
+
+    // MARK: - Private Constants
+
     private let captureSession = AVCaptureSession()
     private var captureDevice: AVCaptureDevice?
     private let captureSessionOutput = AVCapturePhotoOutput()
-    private var didTakePhoto = false
+    private let captureOutput = AVCapturePhotoOutput()
```

Comment:
Do you need both `captureSessionOutput` and `captureOutput`? They look like duplicates to me